### ISSUE TITLE
manual/Home.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#753]
- https://github.com/playframework-ja/translation-project/tree/2.2.5/manual/Home.md
- diffs between [2.2.0/manual/Home.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/Home.md) and [2.2.5/manual/Home.md](https://github.com/playframework-ja/translation-project/tree/2.2.5/manual/Home.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/Home.md 2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.2.5/manual/Home.md    2014-10-06 20:07:20.000000000 +0900
@@ -13,7 +13,7 @@
 1. [[Installing Play|Installing]]
 1. [[Creating a new application|NewApplication]]
 1. [[Anatomy of a Play application|Anatomy]]
-1. [[Using the Play console|PlayConsole ]]
+1. [[Using the Play console|PlayConsole]]
 1. [[Setting up your preferred IDE|IDE]]
 1. [[Sample applications|Samples]]
 1. [Security policy](http://www.playframework.com/code/security)
@@ -58,7 +58,7 @@
 ## Hacking Play

 1. [[Building Play from source|BuildingFromSource]]
-1. [[3rd Party Tooling|ThirdPartyTools]]
+1. [[3rd Party Tools|ThirdPartyTools]]
 1. [[Repositories|Repositories]]
 1. [[Issue tracker|Issues]]
 1. [[Contributor guidelines|Guidelines]]

```

**Comment by garbagetown** [comment]
やります :hand:

**Comment by garbagetown** [comment]
merged at #1041
